src/App.tsx

```diff
@@ -14,11 +16,92 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  selectedGoods: string[],
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedGoods: ['Jam'],
+  };
+
+  title = () => {
+    const { selectedGoods } = this.state;
+
+    let message;
+
+    if (selectedGoods.length === 0) {
+      message = 'No goods selected';
+    }
+
+    if (selectedGoods.length === 1) {
+      message = `${selectedGoods[0]} is selected`;
+    }
+
+    if (selectedGoods.length === 2) {
+      message = `${selectedGoods[0]} and ${selectedGoods[1]} are selected`;
+    }
+
+    if (selectedGoods.length >= 2) {
+      const firstPart = selectedGoods.slice(0, selectedGoods.length);
+
+      message = `${firstPart} are selected`;
+    }
+
+    return message;
+  };
+
+  selecetGood = (item: string) => {
```

Comment:
```suggestion
  addGood = (item: string) => {
```

Reply:
add removeGood method and button - done
good cannot be added twice - done